app/stac_api/migrations/0024_added_update_interval_field.py

```diff
@@ -0,0 +1,132 @@
+# Generated by Django 4.0.10 on 2023-03-01 15:39
+
+import pgtrigger.compiler
+import pgtrigger.migrations
+
+import django.core.validators
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('stac_api', '0023_alter_asset_media_type'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='asset',
+            name='update_interval',
+            field=models.IntegerField(
+                default=-1,
+                help_text=
+                'Interval in seconds in which the asset data is updated.-1 means that the data is not on a regular basis updated.',
+                validators=[django.core.validators.MinValueValidator(-1)]
+            ),
+        ),
+        migrations.AddField(
+            model_name='assetupload',
+            name='update_interval',
+            field=models.IntegerField(
+                default=-1,
+                help_text=
+                'Interval in seconds in which the asset data is updated.-1 means that the data is not on a regular basis updated.This field is can only be set via the API.',
+                validators=[django.core.validators.MinValueValidator(-1)]
+            ),
+        ),
+        migrations.AddField(
+            model_name='collection',
+            name='update_interval',
+            field=models.IntegerField(
+                default=-1,
+                help_text=
+                'Minimal update interval in seconds in which the underlying assets data are updated.',
```

Comment:
Consistency: On asset level the text is "Interval in seconds ...", here it is "Minimal update interval". Probably might make sense to have the same description everywhere, on order to avoid confusion?

Reply:
@hansmannj `update_interval` can only be set on asset level (via the asset upload object).

On item level:
it is and will always be read only. The value is set by pgtriggers. The value is the minimal non negative aggregation of all item's assets or -1 if all assets have -1.

On collection level:
it is and will always be read only. The value is set by pgtriggers. The value is the minimal non negative aggregation of all collection's items or -1 if all items have -1.

Therefore I used different help text because this value is different on asset level as on  item and collection level. This help text is only display on the admin interface and currently the `update_interval` is read only in admin interface.